dataplane/cmd/ctemplate.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	cf "github.com/hortonworks/cb-cli/dataplane/config"
+	ct "github.com/hortonworks/cb-cli/dataplane/ctemplate"
+	fl "github.com/hortonworks/cb-cli/dataplane/flags"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	DataPlaneCommands = append(DataPlaneCommands, cli.Command{
+		Name:  "ctemplate",
```

Comment:
We don't use abbreviations, we shouldn't start doing it now.

Reply:
env is an abbreviation too